tests_python/test_evaluate_expression.py

```diff
@@ -27,6 +27,7 @@ def disable_critical_log():
         yield
 
 
+@pytest.mark.skipif(IS_PY313_OR_GREATER, reason="Crashes on Python 3.13")
```

Comment:
This I'm not happy with either. CPython is throwing random access violations if any of these tests are enabled. I think as a follow up I shoud debug the access violation and log a bug on CPython.

Reply:
I believe the culprit is https://github.com/python/cpython/issues/125723 (these tests have a frame that outlives the scope).

Hopefully it'll be fixed in 3.13.1 (made a change in the commit: https://github.com/fabioz/PyDev.Debugger/commit/e844e4283341111d719bfd35fd5f0298e868891f so that this is 3.13.0)